src/ec/suite_b/public_key.rs

```diff
@@ -28,10 +28,10 @@ use untrusted;
 ///
 /// [NIST SP 800-56A, revision 2]:
 ///     http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-56Ar2.pdf
-pub fn parse_uncompressed_point<'a>(ops: &PublicKeyOps,
-                                    input: untrusted::Input<'a>)
-                                    -> Result<(Elem, Elem),
-                                              error::Unspecified> {
+pub fn parse_uncompressed_point(ops: &PublicKeyOps,
+                                input: untrusted::Input)
+                                -> Result<(Elem, Elem),
+                                          error::Unspecified> {
```

Comment:
I just kind of inherited the convention from BoringSSL, which seems to be "approximately 80" in a way I can't fully describe. Apparently punctuation can overhang by one character? Not sure. I'm looking forward to figuring out how to get rustfmt to format my Rust code in the "wrap to fewest number of lines" style that is used in *ring* and BoringSSL (As opposed to the "one thing per line unless everything fits on one line" style that seems to be the default.)